src/js/timeline.js

```diff
@@ -1,80 +1,69 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Refresh extends React.Component {
+  refreshTimeline() {
+    getTL();
+  }
+  render() {
+    return React.createElement('button', {type: 'button', onClick: this.refreshTimeline}, "Refresh Timeline");
+  }
+}
+
+class Avatar extends React.Component {
+  render() {
+    let user = this.props;
+    return React.createElement('div', {className: 'userInfo'},
+      React.createElement('div', {className: 'img'},
+        React.createElement('img', {src: `${user.profileImageUrl}`, className: 'img-circle'})),
+      React.createElement('div', {className: 'name'}, `${user.name}`),
+      React.createElement('div', {className: 'handle'}, `@${user.twitterHandle}`)
+    );
+  }
+}
+
+class Tweet extends React.Component {
+  render() {
+    let tweet = this.props.tweet;
+    let user = tweet.twitterUser;
+    let date = new Date(tweet.createdAt);
+    let formattedDate = date.toLocaleDateString("en-US", {month:"short", day:"2-digit"});
+    return React.createElement('div', {className: 'tweet'},
+      React.createElement(Avatar, user),
```

Comment:
nice job passing the user object down and not the entire tweet object, exactly what i was looking for